src/app/components/client/ExposureCard.tsx

```diff
@@ -42,77 +42,81 @@ export const ExposureCard = (props: Props) => {
     exposureType,
     exposureDetailsLink,
     dateFound,
-    statusPillContent,
     statusPillType,
   } = props;
 
   const [detailsOpen, setDetailsOpen] = useState(false);
+  const [isMobile, setIsMobile] = useState(false);
+
+  useEffect(() => {
+    const handleResize = () => {
+      const width = window.innerWidth;
+      setIsMobile(width < 768); // $screen-md
+    };
+
+    window.addEventListener('resize', handleResize);
+    handleResize();
+
+    return () => window.removeEventListener('resize', handleResize);
+  }, []);
+
+
+  const date = new Date(dateFound * 1000); // Multiply by 1000 to convert seconds to milliseconds
+
+  const formattedDate = date.toLocaleDateString("en-US", {
+    year: "2-digit",
+    month: "2-digit",
+    day: "2-digit",
+  });
+
 
   const DetailsFoundItem = (props: DetailsFoundProps) => {
     let headline, description;
     if (props.whichExposed === "family") {
       headline = "Family members";
-
-      description = `We found ${props.num} family member`;
-      if (props.num > 1) {
-        description = `We found ${props.num} family members`;
-      }
+      description = `${props.num} found`;
     }
-
     if (props.whichExposed === "email") {
       headline = "Email";
-
-      description = `We found ${props.num} email address`;
-      if (props.num > 1) {
-        description = `We found ${props.num} email addresses`;
-      }
+      description = `${props.num} found`;
     }
 
     if (props.whichExposed === "phone") {
       headline = "Phone number";
-
-      description = `We found ${props.num} phone number`;
-      if (props.num > 1) {
-        description = `We found ${props.num} phone numbers`;
-      }
+      description = `${props.num} found`;
     }
 
     if (props.whichExposed === "address") {
       headline = "Address";
-
-      description = `We found ${props.num} address`;
-      if (props.num > 1) {
-        description = `We found ${props.num} addresses`;
-      }
+      description = `${props.num} found`;
     }
 
     return (
-      <>
+      <dl className={styles.detailsFoundItem}>
         <dt>
           <span className={styles.exposureTypeIcon}>{props.icon}</span>
           {headline}
         </dt>
         <dl>{description}</dl>
```

Comment:
Should this be [`<dd>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dd) (Description _Details_ element) instead of a nested [`<dl>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl) (Description _List_ element)?